lib/jsdom/browser/Window.js

```diff
@@ -1,13 +1,13 @@
 "use strict";
 
 var CSSStyleDeclaration = require("cssstyle").CSSStyleDeclaration;
-var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+var file = require("./file");
```

Comment:
Done but I still had to do some weird things like this because the class need to have the current window/document :
```
this.XMLHttpRequest = this.createXMLHttpRequest(this);
```